ledger/store/trackerdb/sqlitedriver/sql.go

```diff
@@ -131,16 +131,16 @@ func OnlineAccountsInitDbQueries(r db.Queryable) (*onlineAccountsDbQueries, erro
 }
 
 // MakeOnlineAccountsSQLWriter constructs an OnlineAccountsWriter backed by sql queries.
-func MakeOnlineAccountsSQLWriter(tx *sql.Tx, hasAccounts bool) (w *onlineAccountsSQLWriter, err error) {
+func MakeOnlineAccountsSQLWriter(e db.Executable, hasAccounts bool) (w *onlineAccountsSQLWriter, err error) {
 	w = new(onlineAccountsSQLWriter)
 
 	if hasAccounts {
-		w.insertStmt, err = tx.Prepare("INSERT INTO onlineaccounts (address, normalizedonlinebalance, data, updround, votelastvalid) VALUES (?, ?, ?, ?, ?)")
+		w.insertStmt, err = e.Prepare("INSERT INTO onlineaccounts (address, normalizedonlinebalance, data, updround, votelastvalid) VALUES (?, ?, ?, ?, ?)")
 		if err != nil {
 			return
 		}
 
-		w.updateStmt, err = tx.Prepare("UPDATE onlineaccounts SET normalizedonlinebalance = ?, data = ?, updround = ?, votelastvalid =? WHERE rowid = ?")
+		w.updateStmt, err = e.Prepare("UPDATE onlineaccounts SET normalizedonlinebalance = ?, data = ?, updround = ?, votelastvalid =? WHERE rowid = ?")
```

Comment:
Where is this used?

Reply:
it doesn't seem to be used anywhere, it came from the old sql code, I guess at some point onlineaccounts had such thing? 

I removed it since its no longer necesary.